bin/cheritest/cheritest.c

```diff
@@ -105,6 +108,14 @@
 #define	PROT_CHERI_LOCALRET		0
 #endif /* SIGPROT */
 
+#ifdef __mips__
+#define	TRAPNO_CHERI	(T_C2E | T_USER)
+#elif defined(__riscv)
+#define	TRAPNO_CHERI	(EXCP_CHERI)
+#else
+#define	TRAPNO_CHERI	0
```

Comment:
#error instead?

Reply:
I thought about it, but we have some weird fallbacks earlier for SIGPROT and the like to define them all to zero.  We probably should remove all those if we make this an #error?

Reply:
I went with an #error.  I will also remove the SIGPROT stuff as a separate commit in this series.